gnat2goto/driver/ureal_to_binary.adb

```diff
@@ -29,7 +29,7 @@ package body Ureal_To_Binary is
                                               Frac_Part : out String;
                                               Int_End : out Integer;
                                               Frac_End : out Integer)
-     with Pre => (Denominator (Number) > Uint_0);
+     with Pre => (Norm_Den (Number) > Uint_0);
```

Comment:
Difference between `Denominator` and `Norm_Den` is that `Norm_Den` is what you'd normally think of in a rational number, and `Denominator` is a part of GNATs weird "sometimes rational number, sometimes weird exponent notation" thing

Reply:
May be useful, added as a comment.